### PR TITLE
Add dependent field caution in index.rst

### PR DESCRIPTION
**Pull request by gnito-org**

### CODE REVIEW

**Comment by kbond** [comment]
Thanks for the PR but I'm hesitant to add this as is. As I mentioned in the issue, I believe this is a domain issue. If the order in which setters can be called is important, this is a code smell.

I think a note in [this section](https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html#attributes) about how attributes are built would be best. Something like:

"Attributes passed to the `create*` methods are merged with any set via `getDefaults()` and `withAttributes()`."

**Comment by gnito-org** [comment]
I changed the wording.

File: docs/index.rst [hunk 1]
```diff
@@ -187,6 +187,11 @@ This command will generate a ``PostFactory`` class that looks like this:
         }
     }
 
+.. note::
```
Comment (kbond) [inline docs/index.rst:190]: Can you move to the "Attributes" section below (right above the "Faker" title)

**Review by kbond — CHANGES_REQUESTED** [review]

File: docs/index.rst (hunk 1 above)
Comment (gnito-org) [reply docs/index.rst]: Done.

**Comment by kbond** [comment]
Thank you @gnito-org.

**Pull request merged**